wallets/provider-ledger/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import type {
+  Connect,
+  Disconnect,
+  WalletInfo,
+} from '@rango-dev/wallets-shared';
+import type { BlockchainMeta, SignerFactory } from 'rango-types';
+
+import { Networks, WalletTypes } from '@rango-dev/wallets-shared';
+
+import { getLedgerAccounts, getLedgerInstance } from './helpers';
+import signer from './signer';
+
+const WALLET = WalletTypes.LEDGER;
+
+export const config = {
+  type: WALLET,
+};
+
+export const getInstance = getLedgerInstance;
+export const connect: Connect = async () => {
+  const ledgerAccounts = await getLedgerAccounts();
+
+  return ledgerAccounts;
+};
+
+export const disconnect: Disconnect = async ({ instance }) => {
+  instance?.transport?.close?.();
```

Comment:
in your `getInstance` you are returning this: `return { chainId: ETHEREUM_CHAIN_ID };`
I guess it never has `transport` on it, if it's correct so this line, don't do anything. you can remove it.

Reply:
removed it.